setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name='caper',
-    version='1.4.2',
+    version='1.5.0',
```

Comment:
One thought - you could import the version from the caper __init__.py. Then you don't have to change it in two places.

Reply:
Sure, I will think about that. I actually tried it last year with `version.py` (like your `__init__.py` method) but had some issues with `pip`. `setup.py` needed to import the package itself to find `__version__` in `__init__.py`.

https://github.com/ENCODE-DCC/caper/blob/a02e27ace6615e7f8681887743dbbd058ed53be3/caper/version.py

Reply:
Yeah I think you end up having to parse it out of the `__init__.py` without importing the module like you describe. There is some code in `accession`'s `setup.py` to do it: https://github.com/ENCODE-DCC/accession/blob/dev/setup.py#L62

Reply:
Thanks, will take a look and apply that to the next release.